tools/ci_build/github/windows/x64/requirements.txt

```diff
@@ -0,0 +1,11 @@
+numpy==1.21.0
+mypy
+pytest
+setuptools>=41.4.0
+wheel
+git+http://github.com/onnx/onnx.git@be76ca7148396176784ba8733133b9fb1186ea0d#egg=onnx
+protobuf
+sympy==1.1.1
+flatbuffers
+tensorflow==2.8.*
+tf2onnx==1.9.*
```

Comment:
Would be helpful to add a comment explaining why we need tf and tf2onnx for an ORT build. 

That said, generating models outside of the CI and checking them in so we don't add this dependency to builds may be preferable.